readthedocs/search/faceted_search.py

```diff
@@ -20,19 +43,37 @@ def __init__(self, user, **kwargs):
             The `self.user` attribute isn't currently used on the .org,
             but is used on the .com
         """
-        self.user = user
-        self.filter_by_user = kwargs.pop('filter_by_user', None)
-        super().__init__(**kwargs)
+        s = super().search()
+        s = s.source(exclude=['content', 'headers'])
+        # Return 25 results
+        return s[:25]
```

Comment:
I think this limiting result should be done in the consumer end, like in view end. Doing this in here will limit the usage of the `search` api.